tests/extension/oneapi_kernel_compiler_spirv/kernel_compiler_spirv.cpp

```diff
@@ -0,0 +1,235 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Copyright (c) 2023 The Khronos Group Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+*******************************************************************************/
+
+#include "../../common/common.h"
+#include <exception.hpp>
+
+namespace kernel_compiler_spirv::tests {
+
+#ifdef SYCL_EXT_ONEAPI_AUTO_LOCAL_RANGE
+
+sycl::kernel_bundle<sycl::bundle_state::executable> loadKernelsFromFile(
+    sycl::queue& q, std::string file_name) {
```

Comment:
```suggestion
    sycl::queue& q, const std::string& file_name) {
```
since you seem motivated by the `const`-ness safety in this PR. :-)